biggraphite/drivers/elasticsearch.py

```diff
@@ -432,34 +426,19 @@ def insert_points_async(self, metric, datapoints, on_done=None):
         super(_ElasticSearchAccessor, self).insert_points_async(
             metric, datapoints, on_done
         )
-        if metric.name not in self._name_to_metric:
-            self.create_metric(metric)
-
-        datapoints = self.__downsampler.feed(metric, datapoints)
-        if self.__delayed_writer:
-            datapoints = self.__delayed_writer.feed(metric, datapoints)
-        return self.insert_downsampled_points_async(metric, datapoints, on_done)
+        _raise_unsupported()
 
     def insert_downsampled_points_async(self, metric, datapoints, on_done=None):
         """See the real Accessor for a description."""
         super(_ElasticSearchAccessor, self).insert_downsampled_points_async(
             metric, datapoints, on_done
         )
-        if metric.name not in self._name_to_metric:
-            self.create_metric(metric)
-
-        for datapoint in datapoints:
-            timestamp, value, count, stage = datapoint
-            points = self._metric_to_points[(metric.name, stage)]
-            points[timestamp] = (value, count)
-        if on_done:
-            on_done(None)
+        _raise_unsupported()
 
     def drop_all_metrics(self, *args, **kwargs):
         """See the real Accessor for a description."""
         super(_ElasticSearchAccessor, self).drop_all_metrics(*args, **kwargs)
         # Drop indices.
-        self._metric_to_points.clear()
         self._name_to_metric.clear()
```

Comment:
_name_to_metric / _directory_names should be removed too, there are leftovers from MemoryAccessor

Reply:
Done.